projects/dsp-ui/package.json

```diff
@@ -21,7 +21,7 @@
     "@angular/core": "^9.0.0",
     "@angular/material": "^9.0.0",
     "@angular/cdk": "^9.0.0",
-    "@dasch-swiss/dsp-js": "1.0.0-rc.7",
+    "@dasch-swiss/dsp-js": "1.0.0-rc.9",
```

Comment:
No changes in dsp-lib that could have an effect on dsp-ui?

Reply:
I don't believe so. rc.8 added a mock onotology and rc.9 updated the DSP-API version